rustls/src/unbuffered.rs

```diff
@@ -0,0 +1,33 @@
+//! Unbuffered connection API
+//!
+//! This is an alternative to the [`crate::ConnectionCommon`] API that does not internally buffers
+//! TLS nor plaintext data. Instead those buffers are managed by the API user so they have
```

Comment:
small typo: "API that does not internally buffers TLS nor plaintext data" -> "API that does not internally buffer TLS nor plaintext data"